src/VisualStudio/Core/Def/ValueTracking/ValueTrackingToolWindow.cs

```diff
@@ -19,17 +22,29 @@ internal class ValueTrackingToolWindow : ToolWindowPane
     {
         private readonly ValueTrackingRoot _root = new();
 
-        public static ValueTrackingToolWindow? Instance { get; set; }
+        public bool Initialized { get; private set; }
+
+        [MemberNotNullWhen(returnValue: true, nameof(Initialized))]
+        private Workspace? _workspace { get; set; }
```

Comment:
Property and not a field? The naming is a bit strange.

Reply:
It was because `MemberNotNullWhen` can't be applied to fields. I think as noted in https://github.com/dotnet/roslyn/pull/61815#discussion_r921619947 I'm thinking of it backwards